qemu/tests/virtio_mode.py

```diff
@@ -96,9 +96,13 @@ def verify_virtio_mode_guest_linux(session):
 
         :param session: shell Object
         """
-        pci_info = session.cmd_output("lspci")
-        pci_id_pattern = params["pci_id_pattern"]
-        pci_n = re.findall(pci_id_pattern, pci_info)[0]
+        pci_info = session.cmd_output("lspci -n")
+        pci_id_patterns = params.get("pci_id_patterns").split()
+        for pci_id_pattern in pci_id_patterns:
```

Comment:
No need to split any more,  and no need to use the `for`

Reply:
@yanan-fu 
Updated now, thanks!